lib/polly.ml

```diff
@@ -1,3 +1,9 @@
+let assert_eq x y =
+  if x <> y then (
+    Printf.eprintf "0x%x != 0x%x = 0o%o\n%!" x y y ;
```

Comment:
If you feel that using `assert (out = polly_OUT ())` is not enough but prefer to use your own assertion, I would:

* pass a third argument for the location, which is `__LOC__` at the caller
* think libraries should not use stderr but use the exception mechanism to communicate problems
* would use the `Failure` exception with a string argument